util/content.go

```diff
@@ -215,3 +215,38 @@ func GetContent(contentid string, db *gorm.DB, u *User) (Content, error) {
 	}
 	return content, nil
 }
+
+func ConstructDirectoryPath(dir string) (string, error) {
```

Comment:
Copy-paste out of `/api/v1/handlers.go`. In the interest of keeping V1 untouched, I left the old one there. Should we make the refactor on V1, or leave as-is?

Reply:
you can do the refactor on V1 to prevent duplicating code here

Reply:
Good thinking.. Actually this was already refactored in V1 in the mean time - even better!